downloads/pt_PT/github-git-cheat-sheet.md

```diff
@@ -0,0 +1,247 @@
+[http://mac.github.com](http://mac.github.com)
+
+Distribuições de Git para Linux e sistemas POSIX estão disponíveis no site oficial do Git SCM.
+
+### Git para todas as plataformas
+[http://git-scm.com](http://git-scm.com)
+
+## Configurar o Git
+Configurar os detalhes de utilizador ao nível do sistema:
+
+
+```$ git config --global user.name "[nome]"```
+
+Configura o nome que ficará associado aos Git commits
+
+
+```$ git config --global user.email "[endereco-de-email]"```
+
+Configura o email que ficará associado aos Git commits
+
+## Criar repositórios
+Criar um novo repositório ou obter um já existente a partir do URL:
+
+
+```$ git init [nome-do-projeto]```
+
+Cria um novo repositório local com o nome de projecto especificado
+
+
+```$ git clone [url]```
+
+Faz download de um projeto, incluindo toda a sua história
+
+{% endcapture %}
+<div class="col-md-6">
+{{ colOne | markdownify }}
+</div>
+
+{% capture colTwo %}
+
+## Fazer alterações
+Ver estado das alterações e criar um commit
+
+```$ git status```
+
+Lista todos os ficheiros modificados no project atual.
+
+
+```$ git diff```
+
+Mostra as diferenças linha a linha dos ficheiros alterados (pre-stage)
+
+
+```$ git add [ficheiro]```
+
+Classifica um ficheiro como preparado para commit (staged)
+
+
+```$ git diff --staged```
+
+Mostra as diferenças linha a linha dos ficheiros preparados e alterados (pre-stage)
+
+
+```$ git reset [ficheiro]```
+
+Remove o ficheiro da área de preparação (staged), mas preserva o seu conteúdo
+
+
+```$ git commit -m "[mensagem descritiva]"```
+
+Guarda as alterações preparadas permanentemente no histórico de versões
+
+## Branching (história paralela)
+Crie uma versão parelala do histórico de versões para trabalhar em isolamento.
+
+
+```$ git branch```
+
+Lista todos os branches locais no repositório atual
+
+
+```$ git branch [nome-do-branch]```
+
+Cria um novo branch
+
+
+```$ git checkout [nome-do-branch]```
+
+Muda para o branch especificado e atualiza o diretório de trabalho
+
+
+```$ git merge [nome-do-branch]```
+
+Combina a história do branch especificado com o branch atual.
+
+
+```$ git branch -d [nome-do-branch]```
+
+Remove o branch especificado
+{% endcapture %}
+<div class="col-md-6">
+{{ colTwo | markdownify }}
+</div>
+<div class="clearfix"></div>
+
+{% capture colThree %}
+## Alteração de ficheiros
+Mover e remover os ficheiros já versionados
+
+
+```$ git rm [ficheiro]```
+
+Remove o ficheiro do diretório de trabalho e prepara-o para commit (staged)
+
+
+```$ git rm --cached [ficheiro]```
+
+Remove o ficheiro do controlo de versão mas preserva-o no diretório de trabalho
+
+
+```$ git mv [ficheiro-original] [ficheiro-renomeado]```
+
+Muda o nome do ficheiro e o prepara-o para o commit
+
+## Ignorar ficheiros
+Ignore ficheiros e diretórios temporários
+
+```
+*.log
+build/
+temp-*
+```
+
+Um ficheiro `.gitignore` na raiz do projecto, ignora o versionamento acidental de ficheiros e diretórios correspondentes aos padrões especificados
```

Comment:
I would remove the `,` as it separates the subject from the verb and that's grammatically incorrect.